packages/dds/tree/src/test/shared-tree/editing.spec.ts

```diff
@@ -514,7 +514,14 @@ describe("Editing", () => {
 			// Move B before A.
 			tree1.editor.move(rootField, 1, 1, rootField, 0);
 
-			const editor = tree1.editor.valueField({ parent: rootNode, field: brand("foo") });
+			const editor = tree2.editor.valueField({
+				parent: {
+					parent: undefined,
+					parentField: rootFieldKey,
+					parentIndex: 1,
```

Comment:
What's the point of this change?

Reply:
This test was not written properly: both changes were done on `tree1`. Changing this change to happen on `tree2` requires taking into account the fact that the node with the foo field has not moved and is therefore at index 1 instead of index 0 (which is what `rootNode` stands for).